src/docs/product/sentry-basics/tracing/event-detail.mdx

```diff
@@ -64,7 +64,7 @@ The span view may be limited to one project at a time if you're on the [Team pla
 
 #### Embedding and Traversing Transactions
```

Comment:
```suggestion
#### Fetching and Traversing Transactions
```

Reply:
Instead of "Fetching", maybe "Inlining" would be a better word? 

For context, we're inlining (or embedding) transactions onto the span view (waterfall).